org.eclipse.xtext.ide.tests/xtend-gen/org/eclipse/xtext/ide/tests/server/BuildOrderTest.java

```diff
@@ -53,9 +53,10 @@ public void testChain() {
       it.setDependencies(Collections.<String>unmodifiableList(CollectionLiterals.<String>newArrayList()));
     };
     final ProjectDescription d = ObjectExtensions.<ProjectDescription>operator_doubleArrow(_projectDescription_3, _function_3);
-    this.assertEquals(Collections.<ProjectDescription>unmodifiableList(CollectionLiterals.<ProjectDescription>newArrayList(d, c, b, a)), new TopologicalSorter().sortByDependencies(Collections.<ProjectDescription>unmodifiableList(CollectionLiterals.<ProjectDescription>newArrayList(a, b, c, d)), ((Procedure1<ProjectDescription>) (ProjectDescription it) -> {
+    final Procedure1<ProjectDescription> _function_4 = (ProjectDescription it) -> {
```

Comment:
Is there a reason why the compiler now introduces this variable? Is that by intention?